xiblint/rules/simulated_metrics_retina4_0.py

```diff
@@ -2,13 +2,18 @@
 Ensures simulated metrics are for the iPhone SE, which is currently the smallest display profile.
 """
 
+ALLOWED_DEVICES = [
+    'retina4_0',
+    'watch38',
+]
+
 
 def check(context):  # type: (xiblint.xibcontext.XibContext) -> None
     root = context.tree.getroot()
 
     # In Xcode 9 this metadata is in a new place
     device = root.find('device')
-    if device is not None and device.get('id') != 'retina4_0':
+    if device is not None and device.get('id') not in ALLOWED_DEVICES:
```

Comment:
Maybe we should remove the Xcode 8 variant, since it's getting misleading at this point?